core/server-entry.ts

```diff
@@ -59,23 +60,21 @@ function _ssrHydrateSubcomponents (components, store, router, resolve, reject, a
 }
 
 export default async context => {
-  const { app, router, store } = await createApp(context, context.vs && context.vs.config ? context.vs.config : buildTimeConfig)
+  let storeCode = context.vs.storeCode
+  if (config.storeViews.multistore === true) {
+    if (storeCode === undefined) { // this is from url
+      const currentRoute = Object.assign({ path: queryString.parseUrl(context.url).url/* this gets just the url path part */, host: context.server.request.headers.host })
```

Comment:
there's no need for Object.assign when you're creating a new object
```suggestion
      const currentRoute = {
        path: queryString.parseUrl(context.url).url, // this gets just the url path part
        host: context.server.request.headers.host
      }
```